tests/test_synced_collections/synced_collection_test.py

```diff
@@ -678,3 +801,95 @@ def append_value(sl):
             list(executor.map(append_value, [synced_collection] * num_elements))
 
         assert len(synced_collection) == num_elements
+
+    @pytest.mark.skipif(not NUMPY, reason="This test requires the numpy package.")
+    @pytest.mark.parametrize("dtype", NUMPY_INT_TYPES)
+    @pytest.mark.parametrize("shape", (None, (1,), (2,)))
```

Comment:
Do we need to test multi-dimensional shapes? e.g. `(3, 3)`. Since the same shapes are re-used in many tests, it would be nice to specify them once at the top as `NUMPY_SHAPES = (None, (1,), (2,), (3, 3))`.

Reply:
Good ideas.